stdlib/2/textwrap.pyi

```diff
@@ -1,29 +1,57 @@
-from typing import Any, AnyStr
+from typing import AnyStr, List
 
-class _unicode: ...
+class TextWrapper(object):
+    width: int = ...
+    expand_tabs: bool = ...
+    replace_whitespace: bool = ...
+    drop_whitespace: bool = ...
+    initial_indent: str = ...
+    subsequent_indent: str = ...
+    fix_sentence_endings: bool = ...
+    break_long_words: bool = ...
+    break_on_hyphens: bool = ...
 
-class TextWrapper:
-    whitespace_trans = ...  # type: Any
-    unicode_whitespace_trans = ...  # type: Any
-    uspace = ...  # type: Any
-    wordsep_re = ...  # type: Any
-    wordsep_simple_re = ...  # type: Any
-    sentence_end_re = ...  # type: Any
-    width = ...  # type: Any
-    initial_indent = ...  # type: Any
-    subsequent_indent = ...  # type: Any
-    expand_tabs = ...  # type: Any
-    replace_whitespace = ...  # type: Any
-    fix_sentence_endings = ...  # type: Any
-    break_long_words = ...  # type: Any
-    drop_whitespace = ...  # type: Any
-    break_on_hyphens = ...  # type: Any
-    wordsep_re_uni = ...  # type: Any
-    wordsep_simple_re_uni = ...  # type: Any
-    def __init__(self, width=..., initial_indent=..., subsequent_indent=..., expand_tabs=..., replace_whitespace=..., fix_sentence_endings=..., break_long_words=..., drop_whitespace=..., break_on_hyphens=...) -> None: ...
-    def wrap(self, text): ...
-    def fill(self, text): ...
+    def __init__(
+            self,
+            width: int = ...,
+            expand_tabs: bool = ...,
```

Comment:
This is also wrong (the second argument is really `initial_indent`).

Reply:
Fixed. (Same mistake as before -- I was using the order described in the docs). I also double-checked the Python 3 source code and changed the Python 3 stubs to make it match up there as well.